test/09_contracts.ts

```diff
@@ -179,7 +179,10 @@ describe('contracts', () => {
 			expect(dbDeviceTypes).to.have.length(14);
 			expect(newDt).to.not.be.undefined;
 			expect(finDt).to.have.property('name', 'Fin');
-			expect(finDt).to.have.property('contract', JSON.stringify(finDtContract));
+			expect(finDt).to.have.deep.property(
```

Comment:
@mehalter I don't think that you need to chain `.deep` here.
Deep is used if you are passing, eh, a deep property in the property name argument, eg:
```
expect(finDt).to.have.deep.property('contract.length').that.is.a('number')
```
Doesn't cause any issue here though :)